File: Python/Easy/SingleNumber.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        holder = {}
        for i in nums:
            if i in holder:
                holder[i] += 1
            else:
                holder.update({i:1})

        for x, y in holder.items():
            if y == 1:
                return x

File: Java/Easy/SingleNumber.java
class Solution {
    public int singleNumber(int[] nums) {
        HashMap<Integer, Integer> holder = new HashMap<Integer, Integer>();
        int out = 0;
        for (int x : nums){
            if (holder.containsKey(x)){
                holder.put(x,holder.get(x)+1);
            } else {
                holder.put(x,1);
            }
        }
        for (int y : holder.keySet()){
            if (holder.get(y) == 1){
                out = y;
            }
        }
        return out;
    }
}
